index.html

```diff
@@ -9,21 +9,33 @@
     <title>Personal Portfolio Website</title>
 </head>
 
+<!-- Header section -->
+
 <body>
     <header>
         <nav class="navigation">
             <div class="logo">
                 <h3>Dhuruv</h3>
             </div>
             <div class="menu">
-
             </div>
+            <ul class="desktop-menu">
+                <li><a href="#main-sec">Portfolio</a></li>
+                <li><a href="#about-me">About</a></li>
+                <lI><a href="#contact-form">Contact</a></lI>
+            </ul>
         </nav>
     </header>
+
+<!-- Landing Page section -->
+
     <section>
         <div class="headline-section">
             <div class="illustration-mobile">
-                <img src="./icons/header-illsutration-mobile@2x.png" alt="">
+                <!-- Mobile Version Illustraion -->
+                <img class="mobile-version-illustration" src="./icons/header-illsutration-mobile@2x.png" alt="">
+                <!-- Desktop Version Illustraion -->
+                <img class="desktop-version-illustration" src="./desktop-version-files/Header-llustration-desktop@2x.png" alt="desktop-version-illustraion">
             </div>
```

Comment:
Great job @DhuruvSingh 🤝

- [ ] [Optional] You may please try to maintain the Figma design as much as possible. You may reduce the space between the paragraph and the social icons. Also, try to remove the green line just below the `h2` My recent works.
- Your design
![Your design](https://user-images.githubusercontent.com/63917725/177886841-3c3d572c-57c1-40d3-bd83-fb3a16221859.PNG)

Reply:
Sure I will do that sir